node/src/index.ts

```diff
@@ -795,6 +816,10 @@ export class LocalConnection implements Connection {
   async dropTable (name: string): Promise<void> {
     await databaseDropTable.call(this._db, name)
   }
+
+  withMiddleware (middleware: HttpMiddleware): Connection {
+    return this
```

Comment:
We should mention in the docs for `withMiddleware` that it only applies to remote connections.